app/models/author.rb

```diff
@@ -2,6 +2,7 @@ class Author < ApplicationRecord
   def discount
     10
   end
-  has_many :books
+
+  has_many :books, dependent: :destroy
```

Comment:
We tend to prefer using database foreign keys to maintain referential integrity instead of depending on Rails to do it, as the DB has more control and sometime Rails can be bypassed. How would you change this so that the DB automatically cleans up an author's books when the author is deleted?